src/Connection.php

```diff
@@ -450,7 +455,7 @@ protected function runQueryCallback($query, $bindings, Closure $callback)
      */
     protected function getSessionNotFoundMode(): string
     {
-        return $this->config['sessionNotFoundErrorMode'] ?? self::MAINTAIN_SESSION_POOL;
+        return $this->config['sessionNotFoundErrorMode'] ?? self::CLEAR_SESSION_POOL;
```

Comment:
The [README.md](https://github.com/colopl/laravel-spanner/tree/feature/update-laravel-10#session-not-found-exception-handling) should also be modified accordingly.
It is my understanding that the `default` value changes.

Reply:
Thank you. README.md has been fixed.